tests/pathfinding/test_capacity_updates.py

```diff
@@ -14,17 +14,16 @@
 from pathfinding_service.model import TokenNetwork
 from raiden.constants import UINT256_MAX
 from raiden.messages import UpdatePFS
-from raiden.utils import CanonicalIdentifier
+from raiden.utils import CanonicalIdentifier, typing
```

Comment:
Do we want to decide on using either
```
from raiden.utils import typing
```
or 
```
from raiden.utils.typing import ...
```
? Or is there a benefit of doing it on a case by case basis?

Reply:
The problem in this file is that we need both `raiden.utils.typing.TokenNetworkAddress`  and `raiden_libs.types.TokenNetworkAddress`. I'll change the import name to ` raiden_typing`  to make this clearer.

Reply:
Hmm, we could also use `from raiden.utils.typing import TokenNetworkAddress as TokenNetworkAddressBytes`.

Reply:
even nicer